packages/plus/src/components/EstimateCost/OverlayComponent.tsx

```diff
@@ -0,0 +1,161 @@
+import styled from '@emotion/styled'
+import { Icon, Stack } from '@ultraviolet/ui'
+import type { ReactNode } from 'react'
+import { Children, cloneElement, useMemo } from 'react'
+import flattenChildren from 'react-flatten-children'
+import { LineThrough } from './Components/LineThrough'
+import { Strong } from './Components/Strong'
+import { useEstimateCost } from './EstimateCostProvider'
+import { OverlayContextProvider } from './OverlayContext'
+import { ItemResourceName, OverlayRow, StyledBadge } from './componentStyle'
+import { maximumFractionDigits, multiplier } from './constants'
+import type { Units } from './types'
+
+const OverlayContainer = styled.div<{ inView?: boolean }>`
+  position: fixed;
+  z-index: 999;
+  left: 0;
+  right: 0;
+  bottom: ${({ inView }) => (inView ? -120 : 0)}px;
+  height: 120px;
+  background-color: ${({ theme }) => theme.colors.neutral.background};
+  margin: 0 0 0 200px;
+  display: flex;
+  justify-content: center;
+  box-shadow: ${({ inView, theme }) =>
+    inView ? '0' : theme.shadows.defaultShadow};
+  transition:
+    bottom 0.3s,
+    box-shadow 0.3s;
```

Comment:
should we use our theme here ?

Reply:
@philibea I think you're mixing with normal box-shadow? here it's a transition